src/api/libcellml/model.h

```diff
@@ -251,6 +251,16 @@ class LIBCELLML_EXPORT Model: public ComponentEntity, public std::enable_shared_
      */
     bool hasUnresolvedImports();
 
+	/**
+	 * @brief Flatten all imports in this model.
+	 *
+	 * Flatten this model to remove all imports. If any unresolved imports are found
```

Comment:
I think flatten should be changed to actually describe what is going on.  Flatten here has too much of a contextual meaning.